Tools/WinMLDashboard/package.json

```diff
@@ -33,42 +33,42 @@
   },
   "dependencies": {
     "@types/ncp": "^2.0.1",
-    "electron-log": "^2.2.17",
+    "electron-log": "^4.0.7",
     "electron-squirrel-startup": "^1.0.0"
   },
   "devDependencies": {
-    "@types/fs-extra": "^5.0.4",
-    "@types/jest": "^23.3.0",
+    "@types/fs-extra": "^8.1.0",
+    "@types/jest": "^25.1.3",
     "@types/md5": "^2.1.33",
     "@types/node": "^10.5.3",
     "@types/prop-types": "^15.5.4",
     "@types/react": "^16.4.7",
     "@types/react-dom": "^16.0.6",
-    "@types/react-redux": "^6.0.11",
-    "@types/react-select": "^2.0.6",
-    "@types/webdriverio": "^4.13.0",
+    "@types/react-redux": "^7.1.7",
+    "@types/react-select": "^3.0.10",
+    "@types/webdriverio": "^5.0.0",
     "@types/yauzl": "^2.9.0",
     "ajv": "^6.5.2",
-    "babel-minify": "^0.4.3",
+    "babel-minify": "^0.5.1",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "electron": "^3.0.11",
-    "electron-winstaller": "^2.7.0",
-    "fs-extra": "^7.0.1",
+    "electron-winstaller": "^4.0.0",
+    "fs-extra": "^8.1.0",
     "md5": "^2.2.1",
-    "mocha": "^5.2.0",
+    "mocha": "^7.1.0",
     "ncp": "^2.0.0",
     "npm-font-open-sans": "^1.1.0",
     "office-ui-fabric-react": "^6.40.0",
     "react": "^16.5.2",
-    "react-app-rewired": "^1.5.2",
+    "react-app-rewired": "^2.1.5",
```

Comment:
Please use the lower version of react-app-rewired  because of this [https://github.com/timarney/react-app-rewired/issues/358](https://github.com/timarney/react-app-rewired/issues/358)